api/nakadi-event-bus-api.yaml

```diff
@@ -135,13 +117,7 @@ paths:
         - schema-registry-api
       description: Returns a list of all registered `EventType`s
       parameters:
-        - name: X-Flow-Id
-          in: header
-          description: |
-            The flow id of the request, which is written into the logs and passed to called
-            services. Helpful for operational troubleshooting and log analysis.
-          type: string
-          format: flow-id
+        - $ref: '#/parameters/XFlowId'
```

Comment:
Drying up X-Flow-Id makes, but it's incidental to the subscription api. It could be done in a different smaller pr very quickly.